src/test/java/uk/gov/hmcts/darts/dailylist/service/impl/DailyListServiceImpSaveDailyListTest.java

```diff
@@ -75,30 +71,22 @@ void ok_WhenCodeNotMatchExceptionThrown() throws IOException, CourthouseCodeNotM
         when(dailyListRepository.findByUniqueId(anyString())).thenReturn(Optional.empty());
         when(dailyListMapper.createDailyListEntity(
             any(DailyListPostRequest.class),
-            any(CourthouseEntity.class)
+            any(CourthouseEntity.class) // TODO: need to be specific?
```

Comment:
unresolved TODO

Reply:
Will review.

Reply:
Comment removed, existing test that was considering improving but decided not to, as part of this ticket anyway.